recipes/localtileserver/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "localtileserver" %}
+{% set version = "0.3.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6c5c963b0c4c052565af827c7d624c7fd065452ac498b7fb24a102b86baac83c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - large_image_source_gdal
+    - flask >=2.0.0
+    - flask-caching
+    - furl
+    - requests
+    - scooby
+    - ipyleaflet
+    - folium
+
+test:
+  imports:
+    - large_image_source_gdal
+    - localtileserver
+
+about:
+  home: https://github.com/banesullivan/localtileserver
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Local tile server for viewing geospatial raster files with ipyleaflet or folium
+  description: |
+    Need to visualize a rather large (gigabytes) raster you have locally? This is for you. Local tile server for viewing geospatial raster files with ipyleaflet or folium. A Flask application for serving tiles from large raster files in the Slippy Maps standard (i.e., `/zoom/x/y.png`).
+  doc_url: https://github.com/banesullivan/localtileserver/blob/main/README.md
+  dev_url: https://github.com/banesullivan/localtileserver
+
+extra:
+  recipe-maintainers:
+    - banesullivan
```

Comment:
@giswqs, would you like to have a stake in maintaining any of these recipes?

Reply:
Sure. You can add my id to it. I can help maintain them.

Reply:
Thanks! I added you to large_image, large_image_source_gdal, large_image_source_tiff, and localtileserver so that you can help throughout the localtileserver stack if needed. I expect this to require no effort, but just in case I disappear in the wilderness for a year or so, it may be good for you to be able to merge the releases if leafmap is depending on localtileserver